### PR TITLE
Add performance test

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by eile** [comment]
Don't merge: 2b9cd16 needs discussion.

**Comment by eile** [comment]
Retest this please.

File: tests/perf.cpp [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+class Message : public servus::Serializable
+{
+public:
+    Message( const size_t size ) : _bottle( size, '0' ) {}
+    std::string getTypeName() const final { return "zeq::test::Message"; }
+
+private:
+    bool _fromBinary( const void*, const size_t ) final
+    {
+        // BOOST_REQUIRE_EQUAL( size, _bottle.length( ));
```
Comment (tribal-tec) [inline tests/perf.cpp]: rm?

**Comment by tribal-tec** [comment]
+1 otherwise

**Comment by eile** [comment]
Retest this please.

**Pull request merged**